.github/workflows/release.yml

```
@@ -282,22 +282,22 @@ jobs:
         run: |
           $MSIX_VERSION = (Get-Content -Path "apps/flipper/pubspec.yaml" -Raw) -match 'msix_config:\s*([\s\S]*?)\bmsix_version:\s*(\d+\.\d+\.\d+\.\d+)\b' | ForEach-Object { if ($matches.Count -ge 2) { $matches[2] } else { Write-Output "No msix_version found"; exit 1 } }; echo "::set-output name=version::$MSIX_VERSION"
 
-      - name: Upload Artifact
-        uses: actions/upload-artifact@v4  # Fixed this block
-        with:
-          name: windows-build-debug
-          path: apps/flipper/build/windows/x64/runner/Release/flipper_rw.msix
+      - name: Extract msix_version
+        id: get_version
```

actionlint (1.7.4)

step ID "get_version" duplicates. previously defined at line:280,col:13. step ID must be unique within a job. note that step ID is case insensitive

(id)